oneandone/src/main/java/org/apache/jclouds/oneandone/rest/config/OneAndOneProperties.java

```diff
@@ -19,6 +19,7 @@
 public final class OneAndOneProperties {
 
    public static final String POLL_PREDICATE_SERVER = "jclouds.oneandone.rest.predicate.server";
+   public static final String POLL_PREDICATE_SERVER_SUSPENDED = "jclouds.oneandone.rest.predicate.serverdelete";
```

Comment:
oh now I see, it's a predicate to wait for server deletion not really suspension. ok maybe my comment above is still valid for other predicates though

Reply:
I don't quite get what is required here could you please explain more?